### ISSUE TITLE
 Nice to have in layout_in_circle: always plot self edges outside the circle.

### ISSUE THREAD

**Issue by clpippel** [igraph/rigraph#556]
**What is the feature or improvement you would like to see?**
In layout_in_circle: always plot self edges outside the circle.

**Use cases for the feature**
Example.
```
Berger/circle schedule for round robin tournaments
 n <- 32
 n <- max(n - -!(n %%2L), 1)
 r <- c(rbind((seq(from = 0, to = -(n+1)/2+1) %% n) , seq(from=0, to=(n-1)/2)))  # diagonal in Zn
 g <- make_graph(r+1)
  
for (i in seq_len(n-1)) {
   r <- ((r+(n-1)/2) %% n)
   g <- add_edges(g, r+1)
 }
 plot(g, layout=layout_in_circle)
```
For n = 8, the self-edges within the circle are visible.
For larger n, the edges become unrecognizable.
**References**

**Comment by szhorvat** [comment]
I would rephrase this as:  plot self-loops in such a way that they face away from other incident edges. Mathematica does this, see e.g.:

<img width="374" alt="image" src="https://user-images.githubusercontent.com/1212871/181741713-46f55e17-053e-4dec-a528-3c91651d4e9b.png">

This should be done after #407 is fixed.

**Comment by clpippel** [comment]
See [R automated procedure for placement of loops in network graphs](https://stackoverflow.com/questions/76347958/r-automated-procedure-for-placement-of-loops-in-network-graphs) for a workaround using _ggraph_.
